src/handler.rs

```diff
@@ -253,71 +302,35 @@ pub async fn read_order_handler(
     let book: &Book = match ome_state.book(market) {
         Some(b) => b,
         None => {
+            let status: StatusCode = warp::http::StatusCode::NOT_FOUND;
+            let resp_body: OmeResponse = OmeResponse {
+                status: status.as_u16(),
+                message: "Market does not exist".to_string(),
+            };
             return Ok(warp::reply::with_status(
-                "Market does not exist",
-                http::StatusCode::NOT_FOUND,
-            )
-            .into_response());
+                warp::reply::json(&resp_body),
+                status,
+            ));
         }
     };
 
     /* retrieve order */
     let order: ExternalOrder = match book.order(id) {
         Some(o) => o.clone().into(),
         None => {
+            let status: StatusCode = warp::http::StatusCode::NOT_FOUND;
+            let resp_body: OmeResponse = OmeResponse {
+                status: status.as_u16(),
+                message: "Order does not exist in this market".to_string(),
+            };
             return Ok(warp::reply::with_status(
-                "Order does not exist in market",
-                http::StatusCode::NOT_FOUND,
-            )
-            .into_response());
-        }
-    };
-
-    Ok(json(&order).into_response())
-}
-
-/// REST API route handler for updating a single order
-///
-/// Note that this is actually just a convenience method for usability's sake -
-/// there's no such thing as actually mutating an order in-place in the Tracer
-/// protocol (think about how digital signatures work)
-pub async fn update_order_handler(
```

Comment:
Lets make sure we update the docs with this, although I think they are auto generated anyway?

Reply:
Spec will need to be addressed, but otherwise they're automated.